docker/Dockerfile

```diff
@@ -31,7 +30,7 @@ ENV BK_USER=bookkeeper
 # Download Apache Bookkeeper, untar and clean up
 RUN set -x \
     && adduser "${BK_USER}" \
-    && yum install -y java-1.8.0-openjdk-headless wget bash python md5sum sha1sum sudo \
```

Comment:
In the description there contains the reason: 
Also delete yum install for md5sum and sha1sum, since they are already in coreutils, which is default installed;
and left them here will report error: 'No Match for argument: md5sum/sha1sum', while 'yum remove' in docker build.